### PR TITLE
Add border to 'reject' button on room preview card

### PR DESCRIPTION
**Pull request by gefgu**
Signed-off-by: gefgu <gefgu@hotmail.com>

## Checklist

* [ ] Tests written for new code (and old code if feasible)
* [ ] Linter and other CI checks pass
* [X] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

Type: Defect

Notes: Add border to "reject" button on room preview card for clickable area indication. It fixes vector-im/element-web#22623

I used the border design in these download buttons as inspiration: 
download_buttons

Before:
old_buttons
After: 
new_button

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Add border to "reject" button on room preview card for clickable area indication. It fixes vector-im/element-web#22623 ([\#9205](https://github.com/matrix-org/matrix-react-sdk/pull/9205)). Contributed by @gefgu.

### CODE REVIEW

File: res/css/views/rooms/_RoomPreviewCard.pcss [hunk 1]
```diff
@@ -131,5 +131,9 @@ limitations under the License.
             padding: 14px 0;
             flex-grow: 1;
         }
+
+        .mx_AccessibleButton_kind_secondary {
+            border: 2px solid #0dbd8b;
+        }
```
Comment (SimonBrandner) [inline res/css/views/rooms/_RoomPreviewCard.pcss:137]: This should be accomplished by using the correct kind of `AccessibleButton`

**Review by SimonBrandner — COMMENTED** [review]
This will need a sign-off from the design team to make sure this is what we want

File: res/css/views/rooms/_RoomPreviewCard.pcss (hunk 1 above)
Comment (gefgu) [reply res/css/views/rooms/_RoomPreviewCard.pcss:137]: Thanks for the feedback! Can you clarify for me what is the correct kind `AccessibleButton` for use in this case?

Comment (SimonBrandner) [reply res/css/views/rooms/_RoomPreviewCard.pcss:137]: IIRC, `primary_outline` is the one, but you can find all of them in `AccessibleButton.tsx` :)

**Comment by Sanket-777** [comment]
is this issue still open i would like to work on it ?

**Comment by t3chguy** [comment]
@amshakal any chance you could take over review here

**Review by amshakal — APPROVED** [review]
Looks alright to me. Thanks!

File: res/css/views/rooms/_RoomPreviewCard.pcss (hunk 1 above)
Comment (t3chguy) [reply res/css/views/rooms/_RoomPreviewCard.pcss:139]: @gefgu are you able to take a look at this?

Comment (gefgu) [reply res/css/views/rooms/_RoomPreviewCard.pcss]: @t3chguy - made a commit using the `primary_outline` - I think it solves the issue now - thanks for pointing it out :)

**Review by t3chguy — APPROVED** [review]
Thanks for this @gefgu !

**Pull request merged**